coffee-grpc/coffee-grpc-traces-telemetry/pom.xml

```diff
@@ -0,0 +1,38 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>hu.icellmobilsoft.coffee</groupId>
+    <artifactId>coffee-grpc</artifactId>
+    <version>2.5.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>coffee-grpc-traces-telemetry</artifactId>
```

Comment:
ha pontosak akarunk lenni akkor ez is `coffee-grpc-traces-mptelemetry` mint a masik `coffee-grpc-traces-mpopentracing`

Reply:
valoban, atirtam